common.blocks/version-selector/version-selector.js

```diff
@@ -20,15 +20,15 @@ modules.define('version-selector', [ 'i-bem__dom', 'querystring' ], function(pro
                         var ver = encodeURIComponent(qs.parse(location.href).version || '');
 
                         if (!ver) {
-                            ver = JSON.parse(select.dataset.bem);
-                            ver = ver['version-selector'].default.hash;
+                            ver = select.params.default.name;
                         }
 
                         select.setValue(ver);
                     };
 
                     select.domElem.on('change', function(e) {
-                        var val = e.target.value;
+                        var name = e.target.value;
+                        var hash = select._getHashByVersion(name);
```

Comment:
Actually, it can't be `null` because all values are from this selector. I think, I added `null` in a case of someone will call it in the future, but it's a private method, so even in this case, we can suppose version exists.

Seems it's better to remove `null` at all.